common/supplemental/units.xml

```diff
@@ -8,6 +8,15 @@ For terms of use, see http://www.unicode.org/copyright.html
 
 <supplementalData>
     <version number="$Revision$"/>
+    <!--  
+    <unitIdComponents>
```

Comment:
Why is this here if it's commented out? Is it meant for later, or backed out?

Reply:
I'm getting a strange error that I can't figure out. Will be sending out a note, but wanted to get the bulk of this in.